src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1560,6 +1562,98 @@ namespace cryptonote
     return si.available;
   }
   //-----------------------------------------------------------------------------------------------
+  static std::vector<std::string> xx__get_service_nodes_pub_keys_for_height(uint64_t height)
+  {
+      (void)height; // TODO(doyle): Mock function needs to be implemented
+
+      std::vector<std::string> result;
+      const char *XX__PUB_KEYS[] =
```

Comment:
C++11:
```
std::vector<std::string> result = { "blah", "blah", "blah" };

Reply:
Is temp code, but RAII is unnecessary. const chars * will get pulled out as a static string. I think the better would be std::array but that needs to know the size ahead of time instead of deriving it from the initializer list it self so not to my liking.